scripts/convert_stable_diffusion_checkpoint_to_onnx.py

```diff
@@ -221,7 +226,8 @@ def convert_models(model_path: str, output_path: str, opset: int):
         type=int,
         help="The version of the ONNX operator set to use.",
     )
+    parser.add_argument("--half", action="store_true", default=False, help="half-precision")
```

Comment:
Nit: maybe naming it as `--fp16` or `--float16` would be more explicit than `--half`